test/Test_without_transformation.ml

```diff
@@ -114,7 +114,7 @@ let encode_attributes =
       [ JSX.string "& \"" ]
   in
   assert_string (JSX.render component)
-    "<div data-user-path=\"what/the/path\" about=\"&#x27; &lt;\">&amp; \
+    "<div data-user-path=\"what/the/path\" about=\"&apos; &lt;\">&amp; \
```

Comment:
This change is somewhat non-breaking, previosuly we rendered the hex code instead of the html entity, which isn't a big deal but since the rest used html entity I went with it.